lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,9 @@
 <%! from django.utils.translation import ugettext as _ %>
-<section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
 <div class="recent-updates">
   % for index, update in enumerate(visible_updates):
     <article class="updates-article">
       % if not update.get("is_error"):
-        <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
+        <div class="date" id="msg-date-${index}">${update.get("date")}</div>      
```

Comment:
@staubina I think its important that each Update begin with a heading element, especially since they are the first child of an ```<article>``` If we maintain a logical order here, the date would be an ```<h5>```